trillian_admin_api.proto

```diff
@@ -47,6 +48,9 @@ message GetTreeRequest {
 message CreateTreeRequest {
   // Tree to be created. See Tree and CreateTree for more details.
   Tree tree = 1;
+  // Describes how the tree's keys should be generated.
+  // Only needs to be set if tree.private_key is not set.
+  keyspb.Generator key_gen = 2;
```

Comment:
nit: s/key_gen/generator/ ? (Or possibly something else if you take the comment on the generator rename.)